vdev/src/commands/package/msi.rs

```diff
@@ -1,58 +1,83 @@
 use anyhow::Result;
 
+
+
 #[cfg(windows)]
 use {
     crate::{app, util},
+    crate::app::CommandExt,
+    std::ffi::{OsStr},
     std::env,
     std::fs,
     std::iter::once,
-    std::path::Path,
+    std::path::{Path, PathBuf},
+    std::process::Command,
 };
 
+
+// Use the `bash` interpreter included as part of the standard `git` install for our default shell
+// if nothing is specified in the environment.
+#[cfg(windows)]
+const DEFAULT_SHELL: &str = "C:\\Program Files\\Git\\bin\\bash.EXE";
 /// Create a .msi package for Windows
 #[derive(clap::Args, Debug)]
 #[command()]
 pub struct Cli {}
 
 impl Cli {
     pub fn exec(self) -> Result<()> {
+        #[cfg(not(windows))]
+        {
+            println!("Sorry, the package-msi command is not supported on non-Windows platforms. Nothing was performed.");
+        }
         #[cfg(windows)]
         {
             // TODO: wait for other PR to be merged then replace the below line with app::version()
             let archive_version = env::var("VERSION").or_else(|_| util::read_version())?;
 
-            // rm -rf target/msi-x64
-            // cp -R distribution/msi target/msi-x64
+            // Make sure we start with a fresh `target/msi-x64` target directory and
+            // copy the `distribution/msi` directory to `target/msi-x64`
             let msi_x64_dir = Path::new("target").join("msi-x64");
             fs::remove_dir_all(&msi_x64_dir).ok();
             fs::create_dir_all(&msi_x64_dir)?;
             fs::copy("distribution/msi", &msi_x64_dir)?;
 
-            // cp target/artifacts/vector-"${ARCHIVE_VERSION}"-x86_64-pc-windows-msvc.zip target/msi-x64
+
             let artifacts_dir = Path::new("target").join("artifacts");
             let zip_file = format!("vector-{archive_version}-x86_64-pc-windows-msvc.zip");
             fs::copy(artifacts_dir.join(&zip_file), msi_x64_dir.join(&zip_file))?;
 
-            // pushd target/msi-x64
+            // Ensure in the `msi-x64` directory
```

Comment:
```suggestion
            // Ensure we are in the `msi-x64` directory
```

nit

Reply:
Added this on my local branch, will push shortly